src/components/Task/Task.js

```diff
@@ -1,48 +1,113 @@
 import { format } from 'date-fns';
 import React, { useState } from 'react';
+import StyledCheckbox from '../../../styles/Checkbox.styled';
 import StyledListItem from '../../../styles/ListItem.styled';
 import useStore from '../../services/useStore';
 import Button from '../Button/Button';
-import Checkbox from '../Checkbox/Checkbox';
 import Form from '../Form/Form';
 
 export default function Task(props) {
+	const {
+		task: { id, name, done, dueDate },
+	} = props;
+
 	const deleteTask = useStore(state => state.deleteTask);
 	const [editMode, setEditMode] = useState('');
+	const [activeMode, setActiveMode] = useState(true);
+	const [handleSubmit, setHandleSubmit] = useState(false);
```

Comment:
The naming `handleSubmit` is confusing. I would expect a `function` here and not a boolean. Highly recommend renaming this to more descriptive, that a colleague can easily understand, that this variable holds a `bool` value.

Variable that contains `boolean` should start with `is` or `has`
E.g.: `isActive`, `hasActiveMode`, `isInEditMode`, etc.

Reply:
Thanks for the remark. Changed it.